README.md

```diff
@@ -2,54 +2,38 @@
 
 [![Build Status](https://travis-ci.org/nodesource/ncm-cli.svg?branch=master)](https://travis-ci.org/nodesource/ncm-cli)
 
-`ncm-cli` is a command-line tool for NodeSource Certified Modules 2.0, designed to make code quality, security, and compliance a breeze. Generate a custom project report, fetch compliance and security information, manage organizational whitelists, and inspect specific packages in greater detail -- all from the command-line.
+The command-line tool for NodeSource Certified Modules 2.0 — designed to make code quality, security, and compliance a breeze. Generate a custom project report, fetch compliance and security information, manage organizational whitelists, and inspect specific packages in greater detail — all from the command-line.
 
 ## Installation
 
 ```
-npm install -g ncm-cli
+$ npm install -g ncm-cli
 ```
 
 ## Usage
 
 ```
-Usage:
-┌─────────────────────────┐
-│ ncm <command> [options] │
-└─────────────────────────┘
-
- -h, --help    Display help for any command OR this message
- -v, --version Print ncm CLI version
-
-ncm details <module>
-ncm details <module@version>
-
-ncm report
-ncm report <directory>
-  -l, --long
-  -c --compliance
-  -s --security
-  --filter=<value>
-
-ncm signin (interactive)
- -g, --github Sign in via GitHub account
- -G, --google Sign in via Google account
+$ ncm <command> [options]
+```
 
-ncm signout
+```
+$ ncm help <command>
 ```
```

Comment:
@juliangruber How do you feel about this "usage" section brief?

Reply:
With a bit of text that could work, eg https://github.com/zeit/now-cli#usage

Reply:
There's not really any text there? Not sure I understand.

Reply:
Oh you're right, sorry I mistook the installation instructions for usage. Yup they link to a full fledged docs page.

Reply:
Ah good call there though, we don't actually link to ours.